dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Controllers/SessionController.cs

```diff
@@ -0,0 +1,47 @@
+using System.ComponentModel.DataAnnotations;
+using Devon4Net.Infrastructure.Logger.Logging;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Devon4Net.Application.WebAPI.Implementation.Business.SessionManagement.Dto;
+using Devon4Net.Application.WebAPI.Implementation.Business.SessionManagement.Service;
+using LiteDB;
+
+namespace Devon4Net.Application.WebAPI.Implementation.Business.SessionManagement.Controllers
+{
+    /// <summary>
+    /// Session controller
+    /// </summary>
+    [ApiController]
+    [Route("[controller]")]
+    [EnableCors("CorsPolicy")]
+    
+    public class SessionController: ControllerBase
+    {
+        private readonly ISessionService _sessionService;
+        
+        public SessionController( ISessionService sessionService)
+        {
+            _sessionService = sessionService;
+        }
+        
+        
+
+        /// <summary>
+        /// Creates a session
+        /// </summary>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("/session")]
```

Comment:
Zeile 35 könnte man geg. in die Route die SessionId einbinden, Vorrschlag:         [Route("/session/{sessionDto.Id:long}")]

Reply:
the ID of a session only exists after it is created, the ID is automatically incremented so the input values are ignored. I suggest to remove the id from the DTO entirely to avoid confusion.

Reply:
I just want to add that you could change the route to `[Route("/estimation/v1/session")]` to keep the route names consistent.